docs/content/guides/operator/indexer-stack-setup.mdx

```diff
@@ -0,0 +1,399 @@
+
+| Pipeline        | Table           | Approximate size (recommended config) |
+|-----------------|-----------------|--------------------------------------------|
+| balances        | balances        | TBD                                        |
+| object-by-owner | object-by-owner | TBD                                        |
+| object-by-type  | object-by-type  | TBD                                        |
+
+## GraphQL setup
+
+GraphQL instances read data from the indexer and the consistent store. The number of instances is determined by the required read throughput.
+
+### Run command
+
+Each GraphQL instance is run using the same command:
+
+```sh
+$ sui-indexer-alt-graphql rpc \
+    --config CONFIG_FILE \
+    --indexer-config INDEXER_CONFIG_FILE \
+    --bigtable-instance BIGTABLE_INSTANCE \
+    --consistent-store-url CONSISTENT_STORE_URL \
+    --database-url DATABASE_URL \
+    --fullnode-rpc-url FULLNODE_RPC_URL
+```
+
+| CLI param              | Description                                                                        |
+|------------------------|------------------------------------------------------------------------------------|
+| `CONFIG_FILE`          | Path to GraphQL configuration file                                                 |
+| `INDEXER_CONFIG_FILE`  | Path to indexer configuration file<br/>Can be set multiple times; once per indexer |
+| `BIGTABLE_INSTANCE`    | GCP Bigtable instance                                                              |
+| `CONSISTENT_STORE_URL` | URL to [consistent store](#consistent-store-setup) API                             |
+| `DATABASE_URL`         | Postgres database connection string                                                |
+| `FULLNODE_RPC_URL`     | URL to fullnode RPC API                                                            |
+
+
+Example:
+
+```sh
+$ sui-indexer-alt-graphql rpc \
+    --config graphql.toml \
+    --indexer-config events.toml \
+    --indexer-config misc.toml \
+    --indexer-config obj_versions.toml \
+    --indexer-config tx_affected_addresses.toml \
+    --indexer-config tx_affected_objects.toml \
+    --indexer-config tx_calls.toml \
+    --indexer-config tx_kinds.toml \
+    --bigtable-instance kv-store-mainnet \
+    --consistent-store-url https://localhost:7001 \
+    --database-url postgres://username:password@localhost:5432/database \
+    --fullnode-rpc-url https://localhost:9000
+```
+
+### Run config recommendations
+
+| Config TOML                                                                                   | Description                       |
+|-----------------------------------------------------------------------------------------------|-----------------------------------|
+| `graphql.toml` | Default GraphQL API configuration. |
+
+<details>
+<summary>
+`graphql.toml`
+</summary>
+    <ImportContent source="examples/prod-config/graphql.toml" mode="code" />
+</details>
+
+[Indexer Config](#indexer-run-config)
```

Comment:
I wasn't sure what this link is doing dangling here all by its lonesome. Could use a sentence to let the user know why they should click it.